### DISCUSSION TITLE
Prerender pages is triggering serverside

### DISCUSSION THREAD

**Discussion by alexandregiordanelli** [sveltejs/kit#3703] [Q&A]
Hello everyone.. I built a simple page, just html and a load function with console.log(new Date())... I would like to understand why I use prerender on this page and the sveltekit keeps print the date that I put on load function on the server.. I thought it is just a HTML file without call server when I put prerender

**Comment by RykGitProjects** [comment]
Hi!
You probably need
export const csr = false;

Based on:
https://kit.svelte.dev/docs/page-options
